{"name": "test_vwo_login_negative", "status": "broken", "statusDetails": {"message": "RecursionError: maximum recursion depth exceeded", "trace": "setup = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"bbf16233bdd9cc7748c73d710c55f8cf\")>\n\n    @allure.epic(\"vwo Login test\")\n    @allure.feature(\"TC01 vwo App negative test\")\n    @pytest.mark.negative\n    def test_vwo_login_negative(setup):\n        driver=setup\n        login_page=LoginPage(driver)\n        login_page.Login_to_vwo(usr=\"admin@gmail.com\",pwd=\"admin\")\n        time.sleep(5)\n>       error_message=login_page.get_error_msg_text()\n                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n\ntest_vwo_login.py:24: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n..\\Pageobjects\\LoginPage.py:41: in get_error_msg_text\n    return self.get_error_msg_text().text\n           ^^^^^^^^^^^^^^^^^^^^^^^^^\n..\\Pageobjects\\LoginPage.py:41: in get_error_msg_text\n    return self.get_error_msg_text().text\n           ^^^^^^^^^^^^^^^^^^^^^^^^^\nE   RecursionError: maximum recursion depth exceeded\n!!! Recursion detected (same locals & position)"}, "start": 1752349032358, "stop": 1752349037997, "uuid": "59f61355-f83c-4094-9c11-8033daec5ec4", "historyId": "efc6822957c2f7073ee16577998408d9", "testCaseId": "efc6822957c2f7073ee16577998408d9", "fullName": "tests.VWOLogintest.test_vwo_login#test_vwo_login_negative", "labels": [{"name": "epic", "value": "vwo Login test"}, {"name": "feature", "value": "TC01 vwo App negative test"}, {"name": "tag", "value": "negative"}, {"name": "parentSuite", "value": "tests.VWOLogintest"}, {"name": "suite", "value": "test_vwo_login"}, {"name": "host", "value": "DESKTOP-F5PR8ID"}, {"name": "thread", "value": "10552-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.VWOLogintest.test_vwo_login"}]}